docs/mixin/alerts/alerts.yaml

```
@@ -189,4 +189,40 @@ groups:
       severity: warning
     annotations:
       summary: High cache eviction in Promscale
-      description: "Promscale {{ $labels.name }} has a hit ratio of {{ $value }}."
\ No newline at end of file
+      description: "Promscale {{ $labels.name }} has a hit ratio of {{ $value | humanizePercentage }}."
+- name: promscale-database
```

```suggestion
- name: promscale-database-connection
```
nit: wdyt about using `promscale-database` group name for sql alerts?

---

sure, sgtm.